site/projects/intro_timeseries.md

```diff
@@ -11,6 +11,7 @@ Students will learn the basics of Numpy and Matplotlib with an eye towards creat
 Potentially include Pandas (e.g. basic dataframes, block averages, moving window averages)?
 
 ## Related Training Notebooks
+* [Basic Time Series Plotting](https://unidata.github.io/python-training/workshop/Time_Series/basic-time-series-plotting/)
```

Comment:
See comment on `site/projects/plotting_satellite_data.md ` first.

The notebook here has some great material for the "Matplotlib (basics)" and "Matplotlib (intermediate)" training notebooks. In both cases, we would simply create some sample data (two variable), rather than fetching it from the NDBC, and then focus on the plotting. The basics might cover simply plotting the data, changing the color of the line, and, say, the changing up the axis tick marks...adding a title (thinking out loud here) The intermediate could then create some sample data, but use time strings for the independent variable. Then the notebook would pickup where the basics notebook left off, but then start looking at formatting the x axis for dates, add a legend, and maybe some more fancy stuff like shading the timeseries between two dates (like shading the chart using light-grey during the nighttime hours). Not saying you are signed up to do this, so no pressure...just giving you more of a feel for how the links to the python-training website are bigger than the training notebooks. That said, they are still incredibly useful to have listed! As with the satellite page, I'd recommend adding a "Related Comprehensive Notebooks" subsection to hold this link.